src/me/drton/flightplot/export/ExporterConfiguration.java

```diff
@@ -8,6 +8,7 @@
 public class ExporterConfiguration {
     private boolean splitTracksByFlightMode;
     private final static String SPLIT_TRACK_BY_FLIGHT_MODE_SETTING = "splitTracksByFlightMode";
+    private float altOffset = 0;
```

Comment:
Here you also need to add the appropriate preference name (String constant), and serialize/deserialize the value in the methods below (saveConfiguration,loadConfiguration). (e.g. like splitTracksByFlightMode)